lib/utils/colors.dart

```diff
@@ -12,4 +12,9 @@ const primaryGreyDark = 0xFF444444; // Used on title text
 const primaryWhite = 0xFFFFFFFF;
 const backgroundColor = 0xFFFFFFFF; // White
 const transparent = 0x00000000; // White
+const FosforescentGreen = 0xff25df0c;
```

Comment:
Here we have broken the existing style (cap letter)
I want to avoid all global scope vars like this and files like this, I want to use the material theme system to manage app wide colours, for now this is fine to keep, but will need a small refactor to fix.

Reply:
In the new screens we are not using the colors.dart file anymore, so i delete this const for the moment.